skyportal/handlers/api/photometry.py

```diff
@@ -1710,6 +1715,9 @@ def get(self, obj_id):
         outsys = self.get_query_argument('magsys', 'ab')
         include_owner_info = self.get_query_argument('includeOwnerInfo', False)
         include_stream_info = self.get_query_argument('includeStreamInfo', False)
+        include_validation_info = self.get_query_argument(
+            'includeValidationInfo', False
```

Comment:
I don't see the docstring below updated to reflect the new argument.

Reply:
Done.